architecture/additional_concepts/authorization.adoc

```diff
@@ -490,6 +490,19 @@ JSON merge on RBAC resources.
 endif::[]
 ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
 
+ifdef::openshift-enterprise,openshift-origin[]
+[[cluster-role-aggregation]]
+
+=== Cluster Role Aggregation
```

Comment:
We should probably link to the Kube docs so there is at least some example of how to create a new cluster role that gets aggregated into (as apposed to cluster roles that aggregate into other cluster roles).

Reply:
I didn't know you could create a cluster role. If that's true, the steps for creating a cluster role should go in https://docs.openshift.com/container-platform/3.9/admin_guide/manage_rbac.html, not here, and describe the option to create a role that uses aggregation.
Can we talk about this one?